.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+***Exception or Stack Trace***
+Add the exception log and stack trace if available
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+***Code Snippet***
+Add the code snippet that causes the issue.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Setup (please complete the following information):**
+ - OS: [e.g. Android 10]
+ - IDE : [e.g. Android Studio 4.0]
```

Comment:
I don't think this is nearly as important as the version of XCode. I'd remove this field.

Reply:
Disagree as they could be using VS Code, IntelliJ, NetBeans, etc and it's important to know as much of the environment as we can to rule out other things